external/fv3fit/fv3fit/pytorch/graph_loss.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import dataclasses
+
+
+@dataclasses.dataclass
+class LossConfig:
+    """
+    Attributes:
+        loss_type: one of "mse" or "mae"
+    """
+
+    loss_type: str = "mse"
+
+    def __post_init__(self):
+        if self.loss_type not in ("mse", "mae"):
+            raise ValueError(
+                f"loss_type must be 'mse' or 'mae', got '{self.loss_type}'"
+            )
+        """
+        Returns the loss function described by the configuration.
+
+        Args:
+            loss_type: type pf loss function
+        Returns:
+            loss: pytorch loss function
+        """
+        if self.loss_type == "mse":
+            self.loss = torch.nn.MSELoss()
+        elif self.loss_type == "mae":
+            self.loss = torch.nn.L1Loss()
+        else:
+            raise NotImplementedError(f"loss_type {self.loss_type} is not implemented")
+
+    def multi_timestep_loss(
+        self, multistep, train_model, inputs, labels
+    ) -> torch.nn.functional:
+        """
+        Multistep loss fucntion, used during training
+        Args:
+            multistep: number of multistep loss calculation
```

Comment:
```suggestion
            multistep: number of data timesteps to include in loss calculation
```

"multistep" isn't a well-defined term, this should make it more precise.